src/main/java/be/syntra/devshop/DevshopFront/controllers/UserController.java

```diff
@@ -1,18 +1,70 @@
 package be.syntra.devshop.DevshopFront.controllers;
 
+import be.syntra.devshop.DevshopFront.models.StatusNotification;
+import be.syntra.devshop.DevshopFront.models.dtos.PaymentDto;
+import be.syntra.devshop.DevshopFront.services.CartService;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 
 @Slf4j
 @Controller
 @RequestMapping("/users")
 public class UserController {
 
+    private CartService cartService;
+
+    @Autowired
+    public UserController(CartService cartService) {
+        this.cartService = cartService;
+    }
+
+    // todo: remove if no problems during testing
```

Comment:
fix the TODO if this relates to DEV-012. Remove entirely + commented out code, if this is from DEV-013

Reply:
DEV-012 -> removed